pkg/config/remote_config_structs.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+type JVSKeyConfig struct {
```

Comment:
This file itself doesn't seem to imply it's remote. Should we name it as `jvskey_config.go`?

Reply:
I was planning to write all remote config struct in this file. How about `config_structs.go`?

Reply:
IIUC, this config is used for the JVS service. Why not have it in the https://github.com/abcxyz/jvs/blob/main/pkg/config/justification_config.go?

Reply:
done